orm/internal/stablejson/encode.go

```diff
@@ -0,0 +1,93 @@
+package stablejson
+
+import (
+	"bytes"
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protopath"
+	"google.golang.org/protobuf/reflect/protorange"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+// Marshal marshals the provided message to JSON with a stable ordering based
+// on ascending field numbers.
+func Marshal(message proto.Message) ([]byte, error) {
```

Comment:
haven't done a deep check, but maybe there are some protobuf well-known types that need to be json marshalled in a custom way, examples are: timestamp, struct, etc.

Reply:
yes, you're right... hopefully it's not too bad. will take a look soon

Reply:
I think for now, maybe this is okay just because the main use case is to do stable marshaling for tests. I'm mainly trying to prevent spurious CI failures rather than a full fledged encoder